app/User.php

```diff
@@ -18,6 +18,11 @@ class User extends Authenticatable
         'name', 'email', 'password',
     ];
 
+    public function getMobileAttribute($value)
```

Comment:
what is use of this?

Reply:
I was trying it as alternate to session, but it didn't worked.